PowerEditor/src/NppNotification.cpp

```diff
@@ -418,23 +418,23 @@ BOOL Notepad_plus::notify(SCNotification *notification)
 			if (notification->nmhdr.hwndFrom == _statusBar.getHSelf())
 			{
 				LPNMMOUSE lpnm = (LPNMMOUSE)notification;
-				if (lpnm->dwItemSpec == DWORD(STATUSBAR_CUR_POS))
+				if (lpnm->dwItemSpec == DWORD(STATUSBAR_DOC_TYPE))

```

Comment:
Is there a reason why STATUSBAR_DOC_TYPE and STATUSBAR_CUR_POS are swapped?

Reply:
There are 2 commits in this PR. The [first commit](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/5523/commits/5bb118d96e1c516c59435bd13b778b269f52347a) is code reordering for readability, the [second commit](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/5523/commits/9b5a39b58f7844e62e46d4952155c29964c7c6f3) is the actual change.

Reply:
Thanks for the hint: "Sort conditionals like the display order" explains it all :-)